packages/mongo/src/lib/functions/generateMongoDoc.ts

```diff
@@ -0,0 +1,6 @@
+import { Model, model } from 'mongoose';
+import { MongoProvider } from '../MongoProvider';
+
+export function generateMongoDoc(collectionName: string): Model<MongoProvider.DocType> {
```

Comment:
@dan-online do you think this should be moved to `MongoProvider` as a `public static` method? It's not used elsewhere, so I think it would make sense to just attach it to the provider since it's a simple method.